### PR TITLE
Add Docker-Compose Fille update readme and update to 1552

### PR DESCRIPTION
**Pull request by GiroudMathias**

### CODE REVIEW

**Comment by GiroudMathias** [comment]
I have forgot old licensekey on docker-compose.yml

File: README.md [hunk 1]
```diff
@@ -31,9 +31,12 @@ docker run -d \
   -p 30120:30120 \
   -p 30120:30120/udp \
   -v /volumes/fivem:/config \
+  -t \
```
Comment (frebib) [inline README.md]: nit: These are often used together like `  -ti \`
Is the interactive/tty actually required for this?

File: README.md [hunk 2]
```diff
@@ -31,9 +31,12 @@ docker run -d \
...
+  -i \
   spritsail/fivem
 ```
 
+
```
Comment (frebib) [inline README.md]: Extra space that you don't need

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+services:
+  api:
+    image: spritsail/fivem
+    container_name: container-name
+    restart: always
+    stdin_open: true
+    tty: true
+    volumes:
+      - "/path/to/ressources/folder:/config"
```
Comment (frebib) [inline docker-compose.yml]: `s/ressources/resources/`

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  api:
+    image: spritsail/fivem
+    container_name: container-name
```
Comment (frebib) [inline docker-compose.yml]: Perhaps `container_name: fivem`, or just leave it out altogether?
A name is a) personal preference and b) not required to start the container

**Comment by frebib** [comment]
Sorry I didn't reply sooner. I don't even recall getting an email about this PR being opened

File: README.md (hunk 1 above)
Comment (GiroudMathias) [reply README.md]: It's required to prevent SIGSEGV exeption

**Comment by frebib** [comment]
Merged and squashed in 2b0e8c33815573f7e88641ad50ec07eb5858ea61

**Pull request closed without merging**